tools/benchmark/src/main/java/com/datadog/benchmark/ext/FileExt.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.benchmark.ext
+
+import java.io.File
+import java.nio.charset.Charset
+
+/*
+ * The java.lang.File class throws a SecurityException for the following calls:
+ * - canRead()
+ * - canWrite()
+ * - delete()
+ * - exists()
+ * - isFile()
+ * - isDir()
+ * - listFiles(…)
+ * - length()
+ * The following set of extension make sure that every call to those methods
+ * is safeguarded to avoid crashing the customer's app.
+ */
+
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
+private fun <T> File.safeCall(
```

Comment:
do we need these methods? they are originally were developed for the SDK usage, so that exception is not propagated to the client code. But if I get it right, this benchmark tooling is just internal tooling? So we probably don't care?

Reply:
Indeed, the whole file will be removed